### ISSUE TITLE
ActionListBox could use a scroll bar

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Valyrox/lanterna#10]
```
It would be nice to have scrollbar for the ActionListBox if it cannot 
accomodate all the elements in the screen. Currently the window just goes blank 
and the listbox seems to just fill up the entire space over the other 
components(I was filling up the listbox in an action event from one of the 
components on the screen).
```

Original issue reported on code.google.com by `m.vino...@gmail.com` on 21 Jun 2012 at 1:49

**Comment by GoogleCodeExporter** [comment]
```
Yes, that would make perfect sense. I was planning to refactor/redesign the GUI 
components shortly, I'll add this to the list.
```

Original comment by `mab...@gmail.com` on 22 Jun 2012 at 4:00
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `mab...@gmail.com` on 22 Jun 2012 at 4:00

**Comment by GoogleCodeExporter** [comment]
```
This has been fixed now in 2.0 snapshot, please give it a try. If you don't set 
any preferred size on the ActionListBox, it will use available space and add a 
scrollbar if the space isn't sufficient. If you set a preferred size, it will 
display a scrollbar if the list contains more items than the size allows.
```

Original comment by `mab...@gmail.com` on 7 Jul 2012 at 9:02
- Changed state: **Fixed**
